src/utils/helpers/sanitizeText.js

```diff
@@ -0,0 +1,2 @@
+export const sanitizeText = (text) =>
+  text.replaceAll(/<\/?[^>]+(>|$)/gi, "").trim();
```

Comment:
Was curious, what is the regex doing exactly?

Reply:
bhai the regular expression is designed to match HTML tags and their content, and it will remove all HTML tags from the text string when used with the replaceAll method. The .trim() function is then used to remove any leading or trailing white spaces from the resulting text.

Reply:
@hoh-wtag do we really need regex here? isn't there any other way to sanitize text?

Reply:
Bhaia, we can iterate through the string character by character to sanitize it, which I thought was not a good option. Kindly let me know if there are any other better options. Thanks.